auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-tooltip top max-width="450px" light content-class="tooltip">
+    <template v-slot:activator="{ on }">
+      <span v-on="on"><a><u>numbered Benefit Company</u></a></span>
```

Comment:
Why is this an anchor?

Reply:
For inherited styling, but i've updated this and applied the styling in the component via a class 👍 Updated!